Source/Fuse.Reactive.JavaScript/Tests/UX/TreeObservable.Set.ux

```diff
@@ -0,0 +1,29 @@
+<Panel ux:Class="UX.TreeObservable.Set">
+	<JavaScript>
+		var TreeObservable = require("FuseJS/TreeObservable");
+		function Tree() {}
+		Tree.prototype = Object.create(TreeObservable.prototype);
+		let tree = new Tree();
+
+		tree.__fuse_id = 0;
```

Comment:
What's with the explicit setting of `__fuse_id` ? Isn't this automatic, or is it something the model differ does later?

Reply:
These are automatically generated by Model system, yes. It's an ugly workaround for the fact that `Scripting.Object` doesn't guarantee object equality/identity on the Uno side.